bundles/io/org.eclipse.smarthome.io.transport.mdns/OSGI-INF/metatype/org.eclipse.smarthome.io.transport.mdns.internal.MDNSClientImpl.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So far we are not using metadata files for configuring services. Instead we make use of our `ConfigDescription`s which can also be understood by our UIs like for example Paper UI.

Thus, I would suggest to move this configuration into a `ConfigDescription` and reference it like its done for the `NetUtil service for example:
https://github.com/eclipse/smarthome/blob/4e91c13e1be8cef932f39094b785b4dcf182b993/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java#L61-L63

https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/ESH-INF/config/networkinterface.xml

Reply:
I used org.eclipse.smarthome.io.net as pattern. There is also metadata used for configuring the service.

Reply:
I added the config description